LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Chema Balsas
+Copyright (c) 2017 Liferay, Inc.
```

Comment:
I'm fine with this, should this be MIT, though? @silverhook, could you give final approval here?

Reply:
MIT, in general, is perfectly acceptable, but I would like to point you to the internal formal process in place that we have for releasing our own code.

Please open an `Outbound Licensing` ticket in the `FOSS` project of our internal JIRA tracker. This is important, so we have all the analyses and decisions logged in one place.

Reply:
Created: https://issues.liferay.com/browse/FOSS-38